sonar-plugin-api/src/main/java/org/sonar/api/rules/ActiveRule.java

```diff
@@ -120,6 +122,16 @@ public RulePriority getSeverity() {
   }
 
   /**
+   * For internal use
+   * @since 6.6
+   * @deprecated
+   */
+  @Deprecated
+  public RulePriority getOverriddenSeverity() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)